pkg/kv/bulk/sst_batcher.go

```diff
@@ -254,6 +259,7 @@ func MakeStreamSSTBatcher(
 		ingestAll: true,
 		mem:       mem,
 		limiter:   sendLimiter,
+		priority:  admissionpb.NormalPri,
```

Comment:
does this deserve a comment? Maybe something that captures the fact that ingestion as part of C2C does not need to be considered elastic as there is no foreground latency being impacted for now.